drivers/video/video_ctrls.c

```diff
@@ -17,17 +17,52 @@ LOG_MODULE_REGISTER(video_ctrls, CONFIG_VIDEO_LOG_LEVEL);
 
 static inline const char *const *video_get_std_menu_ctrl(uint32_t id)
 {
-	static const char *const camera_power_line_frequency[] = {"Disabled", "50 Hz", "60 Hz",
-								  "Auto", NULL};
-	static const char *const camera_exposure_auto[] = {"Auto Mode", "Manual Mode",
-							   "Shutter Priority Mode",
-							   "Aperture Priority Mode", NULL};
-
```

Comment:
Proposing a slightly different strategy aiming to reduce effort when adding new controls.

Reply:
They are not `static`, so it is less obvious for average C developer like me what happens in such case, I'd have to look at the standard to know, while `static const char *const names[];` is obvious.